docker/Dockerfile

```diff
@@ -34,6 +34,8 @@ ENV PYTHONPATH /opt/osmose-backend
 
 ADD . /opt/osmose-backend/
 
+ARG GIT_VERSION
```

Comment:
Can we make the `git describe` in Dockerfile?

Reply:
This is not possible, as the .git file is already excluded inside the Docker build environment